packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -649,6 +649,22 @@ export class GarbageCollector implements IGarbageCollector {
          */
         this.initializeBaseStateP = new LazyPromise<void>(async () => {
             const currentReferenceTimestampMs = this.runtime.getCurrentReferenceTimestampMs();
+            /**
+             * If there is no current reference timestamp, skip initialization. We need the current timestamp to track
+             * how long objects have been unreferenced and if they can be deleted.
+             *
+             * Note that the only scenario where there is no reference timestamp is when no ops have ever been processed
+             * for this container and it is in read mode. In this scenario, there is no point in running GC anyway
+             * because references in the container do not change without any ops, i.e., there is nothing to collect.
+             */
+            if (currentReferenceTimestampMs === undefined) {
+                // Log an event so we can evaluate how often we run into this scenario.
+                this.mc.logger.sendErrorEvent({
```

Comment:
This should never happen in real code because GC runs with summarize and summarize only happens if there are ops. The reason for making this an error is to know if this is happening and then debug it.
However, I don't want to throw an error and close the container here because it's not a critical failure. Things will work just fine if this happened.

Reply:
Added GC configs. Not sure if there is anything else we can log here.

Reply:
K.  For some reason I thought it was an expected code path but can't think of any cases now, sounds good to leave as Error.